src/components/header/HomeDropdown.jsx

```diff
@@ -0,0 +1,355 @@
+/**
+* Dropdown of the menu 'Home & Living' of the Myntra Website
+* Gives the dropdown menu of the category 'Home & Living' and has various sub-categories
+* @returns the dropdown menu with sub-categories
+* @author Sanjana Rao
+*/
+import * as React from 'react';
+import './appbar.css';
+import Box from '@mui/material/Box';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import IconButton from '@mui/material/IconButton';
+import Tooltip from '@mui/material/Tooltip';
+import { styled } from '@mui/material/styles';
+import { makeStyles } from '@material-ui/core';
+import Grid from '@mui/material/Grid';
+import PropTypes from 'prop-types';
+
+/**
+ * @description Makes use of makeStyles from MUI to generate custom styling to components
+ */
+const useStyles = makeStyles(() => ({
+  catContainerHome: {
+    paddingTop: '1.65em',
+    color: '#282c3f',
+    fontSize: '14px',
+    letterSpacing: '0.5px',
+    fontFamily:
+      'Whitney, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica, Arial, sans-serif',
+    fontWeight: '500',
+    textTransform: 'uppercase',
+    '&:hover ': {
+      borderBottom: '4px solid #f2c210',
+    },
+  },
+}));
+
+/**
+ * @description Makes use of styled from MUI to generate custom styling to Menu items
+ */
+const MenuBold = styled(MenuItem)(() => ({
+  fontSize: '14px',
+  paddingTop: '2px',
+  paddingBottom: '2px',
+  paddingRight: '6px',
+  fontFamily:
+    ' Whitney,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif',
```

Comment:
You can make a font family groups in your css and use them in these css. Instead of writing the whole css family again and again you can use that family instance anywhere.